moped-editor/src/components/LookupSelectComponent.js

```diff
@@ -24,10 +24,22 @@ const LookupSelectComponent = (props) => {
   return (
     <Select
       variant="standard"
-      style={{ minWidth: "8em" }}
       id={props.name}
       value={value || props.defaultValue}
       ref={ref}
+      disableUnderline
+      sx={{
+        minWidth: "8em",
+        padding: "10px",
+        fontSize: ".875rem",
+        // adding a border despite disabling it above because the default border is much lower than other inputs
```

Comment:
<img width="404" alt="Screenshot 2024-07-15 at 4 46 34 PM" src="https://github.com/user-attachments/assets/4e3b9da6-0e6c-4fa8-956f-e283b7923bec">
 the solid dark one is the default underline, i moved it up to match the others